From 1b7d1dfca19316d73ce602bf794339db22c43d4b Mon Sep 17 00:00:00 2001
From: Li Wencheng <liwencheng@phytium.com.cn>
Date: Wed, 7 May 2025 17:22:12 +0800
Subject: [PATCH 34/67] PHYTIUM: net/phytmac: Manage WOL on MAC if PHY supports
 WOL feature

Don't manage WoL on MAC, if PHY sets wol fails, So modify if
statement to handle abnormal scenarios correctly.

Signed-off-by: Li Wencheng <liwencheng@phytium.com.cn>
Signed-off-by: Wang Yinfeng <wangyinfeng@phytium.com.cn>
Change-Id: I54cb5ea7f3a51d1b5c4768c6f15fd7e3bb6502ae

Link: https://gitee.com/phytium_embedded/phytium-linux-kernel/commit/3b966d55109f329286fb6c3da0c57ac4a0fa5bfa
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/phytium/phytmac.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/phytium/phytmac.h b/drivers/net/ethernet/phytium/phytmac.h
index 4d66ae498b88..2c9f2966071c 100644
--- a/drivers/net/ethernet/phytium/phytmac.h
+++ b/drivers/net/ethernet/phytium/phytmac.h
@@ -15,7 +15,7 @@

 #define PHYTMAC_DRV_NAME		"phytium-mac"
 #define PHYTMAC_DRV_DESC		"PHYTIUM Ethernet Driver"
-#define PHYTMAC_DRIVER_VERSION		"1.0.39"
+#define PHYTMAC_DRIVER_VERSION		"1.0.40"
 #define PHYTMAC_DEFAULT_MSG_ENABLE	  \
 		(NETIF_MSG_DRV		| \
 		NETIF_MSG_PROBE	| \
--
2.50.1
